app/src/features/apiClient/screens/apiClient/components/sidebar/components/sidebarListHeader/SidebarListHeader.tsx

```diff
@@ -14,13 +18,13 @@ interface ListHeaderProps {
 export const SidebarListHeader: React.FC<ListHeaderProps> = ({ onAddRecordClick, onSearch, menuItems }) => {
   return (
     <div className="sidebar-list-header">
-      <RQButton
+      {/* <RQButton
         size="small"
         type="transparent"
         icon={<PlusOutlined />}
         onClick={onAddRecordClick}
         className="sidebar-list-header-button"
-      />
+      /> */}
```

Comment:
Lets clear the commented code here and from in other instances as well if not required